rules/windows/builtin/security/win_security_susp_possible_shadow_credentials_added.yml

```diff
@@ -0,0 +1,25 @@
+title: Possible Shadow Credentials Added
+id: f598ea0c-c25a-4f72-a219-50c44411c791
+status: experimental
+description: Detects possible addition of shadow credentials to an active directory object.
+author: Nasreddine Bencherchali (rule), Elastic (idea)
+references:
+    - https://www.elastic.co/guide/en/security/7.17/prebuilt-rule-0-16-1-potential-shadow-credentials-added-to-ad-object.html
+    - https://cyberstoph.org/posts/2022/03/detecting-shadow-credentials/
+    - https://twitter.com/SBousseaden/status/1581300963650187264?
+date: 2022/10/17
+logsource:
+    product: windows
+    service: security
+    definition: The "Audit Directory Service Changes" logging policy must be configured in order to recieve events. Audit events are generated only for objects with configured system access control lists (SACLs), and only when they are accessed in a manner that matches their SACL settings. This policy covers the following events ids - 5136, 5137, 5138, 5139, 5141
```

Comment:
https://twitter.com/_w0rk3r/status/1491840899218825222?s=20&t=msB8D9GZITrmbgoH4AFojQ

Reply:
Yep. As the ref suggests, the idea was inspired by a recent blog, I've read and led back to your rule. That's why I mention it in the ref.

Reply:
Nah, just mentioning because this will not be triggered in user objects unless the SACL is set

Reply:
Yep. Hope it's clear enough in the definition section for future readers.

Reply:
Tbh it doesn't sound that clear for me, but if being generic is needed here, that should be ok

Reply:
Ah, that makes sense. I'll add something like that yes. Thanks